docs/pages/protocol/overview.mdx

```diff
@@ -0,0 +1,49 @@
+# XMTP protocol overview
+
+XMTP is a decentralized messaging protocol that enables secure, end-to-end encrypted communication between blockchain addresses. This section covers the technical foundations and implementation details of how XMTP works under the hood.
+
+## Architecture fundamentals
+
+With XMTP, messages are routed through **[topics](/protocol/topics)**, which are unique addresses that identify conversation channels. Messages are packaged as **[envelope types](/protocol/envelope-types)** that contain the actual message data plus metadata for routing and processing.
+
+The protocol prioritizes **security** through advanced cryptographic techniques, delivering end-to-end encryption for both 1:1 and group conversations. Learn about XMTP's [security properties](/protocol/security) and how [wallet signatures](/protocol/signatures) authenticate users.
+
+## Group messaging mechanics
+
+For group conversations, XMTP introduces sophisticated state management:
+
+- **[Epochs](/protocol/epochs)** represent the cryptographic state of a group at any point in time. Each group operation (like adding members) creates a new epoch with fresh encryption keys.
+
+- **[Intents](/protocol/intents)** provide reliable state management through an internal bookkeeping system that handles retries, crashes, and race conditions when applying group changes.
+
+- **[Cursors](/protocol/cursors)** enable efficient message synchronization by tracking where each client left off when fetching new messages.
+
+## Protocol evolution
+
+XMTP evolves through **[XMTP Improvement Proposals](/protocol/xips)** (XIPs), which are design documents that propose new features and improvements. This governance process ensures systematic and decentralized protocol development.
+
+## Key concepts overview
+
+| Concept | Purpose | Key benefit |
+|---------|---------|-------------|
+| [Security](/protocol/security) | End-to-end encryption | Privacy and message integrity |
+| [Topics](/protocol/topics) | Message routing and addressing | Efficient pub/sub architecture |
+| [Envelope types](/protocol/envelope-types) | Message format and structure | Standardized communication protocol |
+| [Epochs](/protocol/epochs) | Group cryptographic state | Forward secrecy with key rotation |
```

Comment:
```suggestion
| [Epochs](/protocol/epochs) | Group cryptographic state | Key rotation and post-compromise security |
```
This is sort of a nitpick. You get forward secrecy on a per-message level primarily just from the ratcheting that happens for each message within an epoch. Rotating the keys every epoch technically also gives you forward secrecy, just more infrequently, but the biggest benefit is it gives you post-compromise security